src/rds/startDBInstances.js

```diff
@@ -0,0 +1,13 @@
+const AWS = require('aws-sdk');
+const startOneDBInstance = require('./startOneDBInstance');
+
+module.exports = function startDBInstances(arns) {
+  if (arns.length > 0) {
+    console.log("Starting RDS instances...");
+    const arnsMap = arns.map(arn => startOneDBInstance(arn));
+    return Promise.all(arnsMap);
+  } else {
+    console.log("No RDS instances to start... See you the next time...");
+    return [];
```

Comment:
Needs a `Promise.resolve([])' so return types are consistent.

Reply:
WIth 
"return Promise.resolve([]).all(arnsMap);"
I am getting type error "Promise.resolve(...).all is not a function",

Reply:
Apologies, line 11 should read something along the lines of `return Promise.resolve([]);`. This makes sure that each branch of the if statement returns a promise.
See: https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/Promise/resolve

Reply:
Thanks, ramesius. Silly me. Updated. Sorry for a delay- spent last 2 days on the planes..